module/VuFind/src/VuFind/View/Helper/Root/AccountMenu.php

```diff
@@ -52,15 +52,36 @@ public function __construct(protected array $config)
     {
     }
 
+    /**
+     * Get all groups with items to display.
+     *
+     * @return array
+     */
+    public function getMenu(): array
+    {
+        $availableGroups = [];
+        foreach ($this->config ?? $this->getDefaultMenu() as $group) {
```

Comment:
Should we add some logic here to detect outdated configurations -- i.e. MenuItems as the top level? We could, for example, automatically move such configuration into an Account group and add the default Lists group to it. This might be a good idea for backward compatibility, so people who forget to update their yaml file don't break their menus entirely.

Reply:
That would be good. I can't work on it the next two weeks but it shouldn't be a problem since these would be breaking changes and would have to wait for VuFind 11.

Reply:
The `menu.phtml` could also be outdated - should I just add a 'deprecated' tag to `getItems()` instead of removing it?

Reply:
If you think it's easy to do, feel free; if not, I think we can justify the breaking change as long as we note it in the changelog.

Reply:
For backward compatibility, additionally provide menu items from `Account` section when rendering the menu. 

I added this in a commit 2 weeks ago. Forgot to mention it and didn't look at this PR again after I it was accidentally closed.